app/src/main/java/com/orange/ouds/app/ui/components/checkbox/CheckboxDemoScreen.kt

```diff
@@ -54,22 +57,34 @@ fun CheckboxDemoScreen(indeterminate: Boolean = false) = DemoScreen(rememberChec
         }
     ) {
         LightDarkDemo {
-            if (indeterminate) {
-                IndeterminateCheckboxDemo(
-                    state = this@DemoScreen,
-                    onClick = {
-                        toggleableState = when (toggleableState) {
-                            ToggleableState.On -> ToggleableState.Off
-                            ToggleableState.Off -> ToggleableState.Indeterminate
-                            ToggleableState.Indeterminate -> ToggleableState.On
+            Row(
+                modifier = Modifier
+                    .background(OudsTheme.colorScheme.background.primary)
+                    .padding(all = OudsTheme.spaces.fixed.medium)
```

Comment:
I think the modifier can be removed because `fillMaxWidth` is not necessary and if I am not wrong `background` and `padding` are already applied by `LightDarkDemo`.

It also seems that `horizontalArrangement` can be removed because `LightDarkDemo` uses a `Box` with `contentAlignment` set to `Alignment.Center`.

Finally, `Row` could be moved in both `IndeterminateCheckboxDemo` and `CheckboxDemo` because these two methods make the assumption that the container is a `Row`. Same comment for the `Column` in `CheckboxItemDemoScreen`.